targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/device/TOOLCHAIN_GCC_ARM/MK64FN1M0xxx12.ld

```diff
@@ -67,6 +61,16 @@ __heap_size__ = 0x6000;
   #define MBED_APP_SIZE 0x100000
 #endif
 
+#if !defined(MBED_BOOT_STACK_SIZE)
+    #define MBED_BOOT_STACK_SIZE 0x400
+#endif
+
+/* With the RTOS in use, this does not affect the main stack size. The size of
+ * the stack where main runs is determined via the RTOS. */
+__stack_size__ = MBED_BOOT_STACK_SIZE;
+
+__heap_size__ = 0x6000;
```

Comment:
It looks like `__heap_size__` and `HEAP_SIZE` symbols now are unused and can be removed.
It would be enough to define  `STACK_SIZE = MBED_BOOT_STACK_SIZE` or even remove also `STACK_SIZE` and use `MBED_BOOT_STACK_SIZE` instead.